CMakeLists.txt

```diff
@@ -29,8 +29,9 @@ if(VCPKG_TARGET_TRIPLET)
   endif(EXISTS ${PLATFORM_TOOLCHAIN})
 endif(VCPKG_TARGET_TRIPLET)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
-
+if(${VCPKG_TARGET_TRIPLET} MATCHES "-alpine-linux-")
+set(ALPINE "Alpine")
+endif(${VCPKG_TARGET_TRIPLET} MATCHES "-alpine-linux-")
```

Comment:
@maxirmx can you check if this is acceptable usage? Using the `VCPKG_TARGET_TRIPLET` seems like a hack but that was easiest to do...